holoviews/plotting/bokeh/chart.py

```diff
@@ -483,6 +483,8 @@ class SpreadPlot(ElementPlot):
     style_opts = line_properties + fill_properties
     _plot_methods = dict(single='patch')
 
+    _stream_data = False # Plot does not support streaming data
```

Comment:
This was very confusing until I realized this might refer to the *bokeh* use of the word 'streaming'.